examples/rstudio/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM rocker/verse:3.6.1

```

Comment:
This image was created by the Rocker community R docker image effort along with r-base. These images are intended to be more version stable for better reproducibility compared to the r-base image. Looking back at their page [here](https://hub.docker.com/r/rocker/verse) I think I want to use their `rstudio` image instead since this `verse` one is all about tex and publishing which we don't care about in this example. 

Initially, I tried using`r-base` but ran into some issues. I also tried using an ubuntu base and then these instructions [here](https://www.rstudio.com/products/rstudio/download-server/debian-ubuntu/), but that also didn't quite work. Now, that it _should_ be working correctly I can give those another shot.

Reply:
Yes, please add a brief comment to this effect.

The comment above says the `rstudio` image seems better but the code is still `verse`?

Reply:
Thanks! I've updated it.